### PR TITLE
liuyuchen77 term proposal

### PR DESCRIPTION
**Pull request by liuyuchen777**

### CODE REVIEW

File: term_proposal/liuyuchen777/nsd_term_proposal.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Thus, I consider to develop a special network simulator for GR which maybe pretty useful for many researcher in this field.
+## Related work
+In computer network research, network simulation is a technique whereby a software program models the behavior of a network by calculating the interaction between the different network entities (routers, switches, nodes, access points, links etc.).<sup>[3]</sup> There are several popular open-source or commercial network simulators. For example, OPNET, Tetcos and NS2/NS3 (except the last one, others are commercial software).
+There’s also some simulators which are not develop for GR but can be modified to satisfy the basic function. For example, one-simulator, a opportunistic network routing simulator.
+However, as I described in last sector, all of these popular simulator donot well support GR and need additional plug-in to execute simulation. The plug-in are not well compatible with software and cause bugs every time. I believe it’s very emerging to develop an open-source tool for related researchers. 
+## Functions and TODO List
```
Comment (yungyuc) [inline term_proposal/liuyuchen777/nsd_term_proposal.md:10]: Before this point, you were introducing the problem to solve.  It's good write-up, and I agree it fits to my understanding of numerical software.

File: term_proposal/liuyuchen777/nsd_term_proposal.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
++ Network Nodes object
```
Comment (yungyuc) [inline term_proposal/liuyuchen777/nsd_term_proposal.md:12]: You need to plan for the code you need to write and the infrastructure (build system, testing, API, etc.)  I am expecting to see them in this section but they are not here.

From what I am reading here, the project looks like more than one can do within 2 months.  I'd need you to elaborate more about the work you will do, even though you may not fully understand the problem to solve, yet.  That will give me an idea how large the scope is, and provide you suggestions.

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: term_proposal/liuyuchen777/nsd_term_proposal.md (hunk 2 above)
Comment (liuyuchen777) [reply term_proposal/liuyuchen777/nsd_term_proposal.md:12]: I am not very ambitious to complete a full version network routing simulator include all the IEEE 802 series protocols. I consider to concentrate on only several Ad-hoc geographic routing method, and features of these protocols are very simple and concise. And I think I can finish these in two month through rewrite some existing open-source code.
And I will add plan of code and overall infrastructure sections before mid term.

**Comment by yungyuc** [comment]
I am merging now for @liuyuchen777 will consider the scope later.

**Pull request merged**